RioCode-Old/src/main/java/frc/robot/Commands/DriveCommands/AlignWithLeftReef.java

```diff
@@ -59,17 +62,45 @@ public AlignWithLeftReef() {
     @Override
     public void initialize() {
 
-        nearestReefTag = Limelight.getNearestReefTag();
-
-        nearestReefPose = Limelight.getTagPose(nearestReefTag);
+        switch (Utils.getSelectedReefPosition()) {
+            case A:
+                targetReefPose = Robot.getAlliance() == Alliance.Blue ? Limelight.getTagPose(ReefTagIDs.blueReefAB) : Limelight.getTagPose(ReefTagIDs.redReefAB);
+                targetReefTag = Robot.getAlliance() == Alliance.Blue ? ReefTagIDs.blueReefAB : ReefTagIDs.redReefAB;
```

Comment:
Thinking these could definitely be lookup tables (java.util.Map), but that's a small optimization